windowing.py

```diff
@@ -205,3 +205,84 @@ def cropped_np(np_sample, start, end):
     end_index = np.where(nanos == end)
     cropped = [np_sample[:,  start_index[0][0]:end_index[0][0] + 1]]
     return cropped
+
+
+def place_data(np_sample, data_key, data_label, dictionary, positive, window_size, stride):
+    """
+        Description: windows the np_sample (np array) with desired specifications and then
+                     places the windows into dictionary["data_key"] and creates a
+                     corresponding number of labels to dictionary["data_label"]
```

Comment:
can you update the code to avoid in function data mutation?
So, instead return the data and labels as a new dict that you then embed in the calling function?